### ISSUE TITLE
Skype starts offline

### ISSUE THREAD

**Issue by felagund** [sagetarian/skype-wrapper#30]
I am using uptodate skype-wrapper on Ubuntu 11.10. I start it automatically on login via startup applications (that is, skype-wrapper is started automatically and it in turn startes skype - that is how it is supposed to be, right?) . However, Skype starts in offline mode (even though it was online when I logged out beforehand) and I have to manually put it online. I guess this is not intended, or is it?

**Comment by bilderbuchi** [comment]
No it is not (and I haven't encountered that). skype should listen to the mode you set in the messaging menu (via the Available,...,Offline buttons), so maybe it's set there by empathy or so (just a guess).
Also, when Shannon sees this, he will want to know which version of skype and skype-wrapper (and maybe python-sykpe, too) you are currently using. you can find that out by entering `apt-cache policy <packagename>` in a terminal.

**Comment by felagund** [comment]
By uptodate I meant uptodate, i.e. what is in the ppa and ubuntu, but
skype:  2.2.0.35-0oneiric2
skype-wrapper:  0ubuntu5-4.3~oneiric2
python-skype: 1.0.31.0-2

I use Pidgin and it starts online (as set in its preferences).

**Comment by sweah** [comment]
I have encountered this behaviour several times, but not on every startup.

**Comment by grissir** [comment]
I know the problem skype-wrapper needs some time to activate the Messaging Menu and so skype is set to offline... But normally the status is set to online after a few seconds isn't it?

**Comment by felagund** [comment]
Hm, I do not think so but will wait some time before putting skype online manually next time I log in.

**Comment by grissir** [comment]
Ok also look how long it takes ro start the messaging menu please

**Comment by felagund** [comment]
I do not know how to find out how long it takes the messaging menu to start.

However, I logged in 10 minutes ago and skype is still offline.

**Comment by grissir** [comment]
Ok then it has to be a bug

**Comment by grissir** [comment]
I was able to reproduce it, I#m not able to change my status via the messaging menu till I set skype to online

**Comment by felagund** [comment]
Now that is strange - I can actually put skype online by setting my online presence to online (even if the presence already had been "online").

**Comment by felagund** [comment]
Ha, I think I found what is causing this. I was using pidgin as my im client (set to autostart) and did not have empathy installed. Now I switched back to empathy and Skype starts online now? Can anybody confirm this?

**Comment by grissir** [comment]
can't reproduce it, but for me: nothing is on nor pidgin neither empathy then skype starts offline else it starts with the current setting of pidgin/empathy

**Comment by felagund** [comment]
Still an issue now when using pidgin as IM and not Empathy.

**Comment by grissir** [comment]
We know about the issue but currently nobody is developing skype-wrapper
If you know someone who is able to continue this project he is welcome

**Comment by sagetarian** [comment]
yeah, the best thing about github is anybody can help out. i've been swamped down with deadlines, overtime and preparing to move to another city, so i encourage collaboration and will infact ensure that full credit to collaborators be given in the readme file. as soon as i have free time i'll try and rock some more bug fixes and less new features.

that being said skype-wrapper will have to look into being its own indicator-messages service provider or at least utilizing pidgin, telepathy, emesene dbus's .. but i'd prefer any new changes to online presence to be totally seperate from another IM, including pidgin .. so therefore we'll need to make changes to indicator-messages and push the changes upstream to that project in order to register skype-wrapper as a valid messaging service for indicator-messages

**Comment by felagund** [comment]
I found out that when I use pidgin, it starts offline, but when I suspend and resume, it goes online. If that helps if anybody wanted to look into this.

**Comment by stonerl** [comment]
What do you mean by suspend and resume? Your Computer? Btw we moved the code and everything else to: https://launchpad.net/skype-wrapper

**Comment by felagund** [comment]
Yes, my computer.  I start my computer, pidgin is online, skype is offline. I suspend and resume my computer, pidgin is online, skype is online. 

Should I open a bug on launchpad then?

**Comment by grissir** [comment]
Bug on lauchpad would be great

**Comment by felagund** [comment]
Done. https://bugs.launchpad.net/skype-wrapper/+bug/1045768
